src/popover.css

```diff
@@ -28,32 +27,35 @@
   inset: auto;
 }
 
-/* This older `:open` pseudo selector is deprecated and support will be removed
-in a later release. */
+/* This older `:open` pseudo selector is deprecated
+   and support will be removed in a later release. */
 @supports selector([popover]:open) {
+  [popover]:not(:open) {
```

Comment:
`@supports selector([popover]:not(:open))` ??? you test one selector then you apply a more complex one :thinking: